models/stubs/index.js

```diff
@@ -8,20 +8,22 @@ module.exports = {
   }),
   createCampaign: fixedData => ({
     name: faker.lorem.words(2),
-    slug: faker.lorem
-      .words(2)
-      .toLowerCase()
-      .split(' ')
-      .join('-'),
-    ...fixedData
+    slug: faker.random.uuid()
   }),
   createActions: (fixedData, amount = 2) => {
-    return Array(amount).fill({
```

Comment:
Just out of curiosity what was the problem with this approach that you change to the for loop?

Reply:
This was causing an error given slug need to be unique. `Array.fill` passes the same values to every element of the Array